### PR TITLE
Editing docs for View functions from index.md to getInnerHtml.md

### PR DESCRIPTION
**Pull request by letitia**
Views

@saponifi3d

### CODE REVIEW

File: _view/attachChildViews.md [hunk 1]
```diff
@@ -3,4 +3,4 @@ header: attachChildViews
 example: View.attachChildViews()
 ---
 
-Removes all child views, then looks at all of the child views that are expected and attaches each view.
+Removes all child views, then looks at all of the expected child views and attaches each view.
```
Comment (saponifi3d) [inline _view/attachChildViews.md]: how about...

Removes all the child views, finds all instances of the views child views, and calls [attach](#attach) on them.

? i feel like the "looks at all of the expected child views" isn't so great (on my part). do you think the above makes it a bit easier? and less of "but how does it look for them?"-y

File: _view/attachOrRender.md [hunk 2]
```diff
@@ -3,6 +3,4 @@ header: attachOrRender
 example: View.attachOrRender(element, parentView)
 ---
 
-This is invoked when View is hydrating the [childViews](/#childViews).  This will inspect the element and check to see if the view is lazy loaded or already attached.  Hydration is the step where the HTML has been loaded by the server, or a view is creating an instance of a child view using a [template adapter](/template-adapters).
-
-If neither of those are true, then it will see if the `data-render` attribute is set on the element. If it is, it will invoke the [render](#render) function.  If not, it will call the [attach](#attach) function on the element.
+This is invoked when View is [hydrating](/fetcher/#hydrate) the [childViews](#childViews).  This inspects the element to see if the view is lazy loaded or already attached.  If neither of those are true, then it will check if the `data-render` attribute is set on the element.  If it is, it will invoke the [render](#render) function.  Otherwise it will call the [attach](#attach) function on the element.
```
Comment (saponifi3d) [inline _view/attachOrRender.md]: If it is, it will invoke 
--->
If so, then it will invoke

sound better?

File: _view/decorateTemplateData.md [hunk 3]
```diff
@@ -3,6 +3,6 @@ header: decorateTemplateData
 example: View.decorateTemplateData(data)
 ---
 
-This function takes the result from [getTemplateData](#getTemplateData) and applies special properties to the data for the view.  The special properties are `_app`, `_view`, `_model`, and/or `_collection`.  These special properties are the instance variables for each of the options set on the view.
+This function takes the result from [getTemplateData](#getTemplateData) and applies special properties to the data for the view.  The special properties are `\_app`, `\_view`, `\_model`, and/or `\_collection`:  the instance variables for each of the options set on the view.
```
Comment (saponifi3d) [inline _view/decorateTemplateData.md:6]: interesting that \ works. YeY!

File: _view/attachOrRender.md (hunk 2 above)
Comment (letitia) [inline _view/attachOrRender.md]: :+1:

File: _view/attachChildViews.md (hunk 1 above)
Comment (letitia) [inline _view/attachChildViews.md]: Correct me if I'm wrong...  `attachChildViews` actually looks at the view's HTML generated by the server, finds the child views in there, and then sets them on the view as `childViews`.  Then it attaches each one of those child views.  Yes?

Comment (saponifi3d) [inline _view/attachChildViews.md]: yeah - it checks the DOM (the filtered to just the $el) for an element with `[data-view]` which is attached by the templateAdapter view helper.

Comment (letitia) [inline _view/attachChildViews.md]: How about

Removes all of the view's current child views, then finds all instances of child views in the generated view HTML and calls `attach` on each child view.

Comment (saponifi3d) [inline _view/attachChildViews.md]: :+1:

**Pull request merged**